tests/Images/Input/Tiff/BigTiff/BigTIFFSamples.html

```diff
@@ -0,0 +1,239 @@
+<html>
```

Comment:
I'd much prefer Markdown if this file is absolutely required. Makes it readable in the repository then.

Reply:
Done. Converted to MD.